ci-operator/step-registry/rosa/sts/account-roles/create/rosa-sts-account-roles-create-commands.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
+ACCOUNT_ROLES_PREFIX=${ACCOUNT_ROLES_PREFIX:-"ManagedOpenshift"}
+CLOUD_PROVIDER_REGION=${CLOUD_PROVIDER_REGION:-"us-east-2"}
+
+# Configure aws
+AWSCRED="${CLUSTER_PROFILE_DIR}/.awscred"
+if [[ -f "${AWSCRED}" ]]; then
+  AWS_ACCESS_KEY_ID=$(cat "${AWSCRED}" | grep aws_access_key_id | tr -d ' ' | cut -d '=' -f 2)
+  AWS_SECRET_ACCESS_KEY=$(cat "${AWSCRED}" | grep aws_secret_access_key | tr -d ' ' | cut -d '=' -f 2)
+
+  aws configure set aws_access_key_id  ${AWS_ACCESS_KEY_ID}
+  aws configure set aws_secret_access_key ${AWS_SECRET_ACCESS_KEY}
+  aws configure set default.region ${CLOUD_PROVIDER_REGION}
```

Comment:
prow ci will be responsible for selecting slice quota region, generally users should not overwrite it to another unexpected region.